configurations/arm/ubuntu2204.yaml

```diff
@@ -0,0 +1,8 @@
+ami_db_scylla_user: 'ubuntu'
+ami_id_db_scylla: 'ami-022c8ce295ce9ac4c'  # ubuntu/images/hvm-ssd/ubuntu-jammy-22.04-arm64-server-20220506
+region_name: 'eu-west-1'
+instance_type_db: 'r6gd.large'
```

Comment:
Why do we use this instance type?
It's not the recommended instance type for scylla.
If we do it in other OS versions, please change that as well.
Please use: 
**im4gn.xlarge**

Reply:
Fixed